content/en/database_monitoring/connect_dbm_and_apm.md

```diff
@@ -158,6 +161,76 @@ func main() {
 
 {{% /tab %}}
 
+{{% tab "v2" }}
+
+Update your app dependencies to include [dd-trace-go@v2.0.0][1] or greater:
+```
+go get github.com/DataDog/dd-trace-go/v2
+```
+
+Update your code to import the `contrib/database/sql` package:
+```go
+import (
+   "database/sql"
+   "github.com/DataDog/dd-trace-go/v2/ddtrace/tracer"
+   sqltrace "github.com/DataDog/dd-trace-go/contrib/database/sql/v2"
+)
+```
+
+Enable the database monitoring propagation feature using one of the following methods:
+1. Env variable:
+   `DD_DBM_PROPAGATION_MODE=full`
+
+2. Using code during the driver registration:
+   ```go
+   sqltrace.Register("postgres", &pq.Driver{}, sqltrace.WithDBMPropagation(tracer.DBMPropagationModeFull), sqltrace.WithService("my-db-service"))
+   ```
+
+3. Using code on `sqltrace.Open`:
```

Comment:
I will change the steps to be unordered! But in terms of adding more comments, I'm afraid we don't actually own this part of the documentation; these lines of code got added since I was recreating existing documentation for v2 (don't worry, this redundancy has been removed now). We could reach out to the DBM team to see if they are able to add more explanations to the docs here.